tests/scripts/quiet/quiet.sh

```diff
@@ -0,0 +1,75 @@
+# -*-mode: sh; sh-shell: bash -*-
```

Comment:
Sorry for not noticing this earlier but there's an inconsistency:

* If `quiet.sh` is executable, it should have a shebang.
* If `quiet.sh` is not executable, it should not have the execute flag and it should be included with `.` rather than executed.

The current code works thanks to bash's fallback behavior: `execve("path/to/quiet.sh", ...)` returns ENOEXEC, so bash falls back to running the script itself. It works but it's surprising and could trip up during maintenance.